### ISSUE TITLE
Allow Modules to get CPU average

### ISSUE THREAD

**Issue by bretg** [prebid/prebid-server#3115]
We're building an internal module that will monitor the Prebid Server CPU usage and start rejecting traffic early in the pipeline once a certain threshold has been reached.

The module will be internal to Magnite because it's going to incorporate some rubicon bidder madness into the logic, but there are two pieces that we're planning to open source. The first of these pieces is the ability for modules to be able to obtain the recent average CPU utilization. By opening this function, others will be able to build similar modules. 

Here's the proposal:

1. Two new host-level configurations allow host companies to control the feature

```
server.cpu-load-monitoring.enabled: true
server.cpu-load-monitoring.measurement-interval-ms: 60000
```

2. A new function becomes available to module writers

```
cpuLoadAverageStats.getCpuLoadAverage();  // returns a float
```

That's it. The module can call this and make decisions about whether to reject the request. Community feedback is welcome.

Will open a different issue on the second open enhancement - the ability to support module-level DB config.

**Comment by bretg** [comment]
Discussed in committee- apparently it's already possible to get CPU utilization from Go modules. Flagging this as Java-only.

**Comment by bretg** [comment]
Released with PBS-Java 2.0
